recipes/djinni-support-lib/all/conanfile.py

```diff
@@ -90,9 +90,7 @@ def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
 
     def package_info(self):
-        #self.cpp_info.libdirs = ["lib"] # thats default anyway
         self.cpp_info.libs = tools.collect_libs(self)
-        self.cpp_info.includedirs = ["include",]
         self.cpp_info.includedirs.append("include/djinni")
```

Comment:
I don't see any typos from my side 🤞 

```suggestion
        self.cpp_info.includedirs.append(os.path.join("include", "djinni"))
```

Reply:
I think I resolved that already with edits of lowercase the option

Reply:
actually, this path is not needed at all, this lib is only included by auto generated code (see  #5059)  , and this always  includes the full path. 
Old versions of the generator did not do that, for them is the part that should not exist, and I start to regret having added them :-) 
But now I have a commit that fixes that by removing "include/djinni" at all